genai-perf/genai_perf/record/record.py

```diff
@@ -15,13 +15,23 @@
 import importlib
 import os
 from abc import ABCMeta, abstractmethod
+from dataclasses import dataclass
 from statistics import mean
 from typing import Dict, Union
 
 from genai_perf.exceptions import GenAIPerfException
 from genai_perf.types import RecordValue
 
 
+@dataclass(frozen=True)
+class ReductionFactor:
+    NS_TO_MS = 6
+    NJ_TO_MJ = 6
+    B_TO_GB = 9
+    PERCENTAGE = -2
```

Comment:
Is the negative intentional here?
It doesnt follow the pattern of the rest of them, so I figured i would check.

Reply:
Yes, in this case we are going from something like 0.34 to 34% (so it's an anti-reduction!)